src/ai/backend/manager/models/storage.py

```diff
@@ -199,14 +200,18 @@ class StorageVolume(graphene.ObjectType):
     class Meta:
         interfaces = (Item,)
 
-    # id: {proxy_name}:{name}
+    # id = graphene.String()  # "{proxy}:{name}""
     backend = graphene.String()
     fsprefix = graphene.String()
     path = graphene.String()
     capabilities = graphene.List(graphene.String)
     hardware_metadata = graphene.JSONString()
     performance_metric = graphene.JSONString()
     usage = graphene.JSONString()
+    proxy = graphene.String(
+        description=description_msg("24.03.1", "Name of the proxy which this volume belongs to.")
+    )
+    name = graphene.String(description=description_msg("24.03.1", "Name of the storage."))
```

Comment:
Shouldn't the introduced version be `24.03.0`?

Reply:
I updated it. thank you!